### PR TITLE
Support for MQTT Configuration in a single Backlog command

### PR DESCRIPTION
**Pull request by deveth0**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -3,3 +3,20 @@
...
+    command: "Backlog"
+    value: >
+      MqttHost {{ tasmota_mqtt_host }};
+      MqttPort {{ tasmota_mqtt_port }};
+      MqttUser {{ tasmota_mqtt_user }};
+      MqttPassword {{ tasmota_mqtt_password }};
+      MqttClient {{ tasmota_mqtt_client }};
+      MqttTopic {{ tasmota_mqtt_topic }};
+      MqttFullTopic {{ tasmota_mqtt_fulltopic }};
+  when: >
```
Comment (tobias-richter) [inline tasks/main.yml:19]: You can use the list syntax here:
```
when:
  - tasmota_mqtt_user
  - tasmota_mqtt_password
  - tasmota_mqtt_host 
  - tasmota_mqtt_port
  - tasmota_mqtt_client 
  - tasmota_mqtt_topic 
  - tasmota_mqtt_fulltopic
```

**Review by tobias-richter — CHANGES_REQUESTED** [review]
Not everyone wants to set client, topic and fulltopic and keep the defaults.
IMHO this is to specific for a special use-case.
From my point of view only host, port are common for everyone. There may be setups out there which do not use user and passwort for their MQTT server (even if it is bad).

Since the Backlog functionality is already supported in the current role and the user can already configure the values in one command I do not see a need in making this part of the role.
Perhaps an example in the example section would make sense.

**Comment by deveth0** [comment]
If you don't set values for those fields, they are not set, so I think it's fine to support the full set and only set those fields that are configured.

As mqtt is a basic functionality of tasmota, it justifies a build in block imho.

**Comment by tobias-richter** [comment]
@deveth0 In my setup I have not set these values and i was not required to. I am only setting `FriendlyName1` and `DeviceName` and enabling the home assistant integration and it works like a charm ootb without setting topic and fulltopic. If I now want to adjust the basic mqtt settings based on your implementation I have to lookup the automatically configured values and port them to the configuration management. 
From my point of view this would be a little bit awkward.

**Comment by deveth0** [comment]
Nope, cause the default is an empty string which does not cause a change (using commands without value are just read commands for the current value).
So if your want to set the host, you only set the host value and leave the others.

**Comment by tobias-richter** [comment]
Let's have a chat about this next week. I still see no need for this since it does not offer new functionality. Everything is already possible with the existing implementation and it is simply an new command entry that is passed to the action module.

Beside that it will not report a valid changed/not changed response.

**Review by tobias-richter — APPROVED** [review]

**Pull request merged**